healthcheck.py

```
@@ -0,0 +1,50 @@
+VERSION = "0.1.0"
+
+
+class HealthHandler(BaseHTTPRequestHandler):
+    """Handles GET /health requests."""
+
+    def do_GET(self):
+        if self.path == "/health":
+            response = {
+                "status": "ok",
+                "version": VERSION,
+                "timestamp": str(datetime.now()),
```

Ruff (0.15.2)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)